ui/js/models/perf/alerts.js

```diff
@@ -80,8 +80,19 @@ treeherder.factory('PhAlerts', [
         });
         AlertSummary.prototype.getTextualSummary = function(copySummary) {
             var resultStr = "";
-            var improved = _.filter(this.alerts, function(alert) {return !alert.is_regression && alert.visible;});
-            var regressed = _.filter(this.alerts, function(alert) {return alert.is_regression && alert.visible;});
+            var improved = _.sortBy(_.filter(this.alerts, function(alert) {
+                return !alert.is_regression && alert.visible;}),
+            'amount_pct').reverse();
+            var regressed = _.sortBy(_.filter(this.alerts, function(alert) {
+                return alert.is_regression && alert.visible;}),
+            'amount_pct').reverse();
+
+            var formatAlert = function(alert, alertList){
+                return _.padLeft(alert.amount_pct.toFixed(0), 3) + "%  " +
+                _.padRight(alert.title, _.max(alertList, function(alert){ return alert.title.length; }).title.length +5) +
```

Comment:
This is O(n^2) code (since formatAlert is called n times, and it calls _.padRight n times) which is not performant. However, n is so small in this case that optimization isn't necessary. Just wanted to call that out, in case you didn't see this yourself. :)